pages/[countryCode]/[lang]/[product].tsx

```diff
@@ -97,7 +107,7 @@ const ProductPage: FunctionComponent<Props> = ({
             buildLanguages={buildLanguages}
             countries={countries}
           >
-            <ItemContainer>
+            <LineItemsContainer>
```

Comment:
You must remove `LineItemsContainer` here. You don't need it.

Reply:
Ahh, grazie!